section 4_Flask_Restfull/app.py

```diff
@@ -1,46 +1,66 @@
-from flask import Flask,request
-from flask_restful import Resource, Api,reqparse
-from flask_jwt import JWT,jwt_required
+'''This is the main file which contains all the end points'''
+
+
+
+from flask import Flask
+from flask_restful import Resource, Api, reqparse
+from flask_jwt import JWT, jwt_required
 from security import authenticate, identity
 
 app = Flask(__name__)
-app.secret_key = 'jose'
+app.secret_key = 'sdfdslfls@#1223$lskd'
 api = Api(app)
 
-jwt = JWT(app,authenticate,identity) # /auth end point dega
+jwt = JWT(app,authenticate, identity) # gives /auth end point
 
-items = []
+items: list = []
 
 
 class Item(Resource):
     parser = reqparse.RequestParser()
-    parser.add_argument('price', type=float,required=True, help='this field cannot be left blank')
+    parser.add_argument('price', type = float, required = True, help = 'this field cannot be left blank')
```

Comment:
As per pep 8, no space should be added around = when using kwarg or default value with an arg.